variables-cluster.tf

```diff
@@ -82,8 +82,8 @@ variable "image_signing_keys" {
 }
 
 variable "load_balancers" {
-  default     = "public"
-  description = "The type of subnets to create for load balancers."
+  default     = "internal"
```

Comment:
That's not the default on OKE. If you set it to internal and a user creates a service of type load balancer without the overriding annotation, this will fail.

Please change to `both` or `public`

Reply:
The `internal` value aligns with `preferred_load_balancer=internal` - one of the parameters we discussed for minimized exposure by default. We can leave the existing value to match the service, though it's a bit of a departure from other changes. `both` or `public` means creating the `pub_lb` subnet/NSG by default as well.

We could also address with a note on the variable description + examples. What do you think?

Reply:
Retained both/public LB values as discussed to match CCM defaults.